packages/amplify_auth_cognito/android/src/test/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AmplifyAuthCognitoPluginTest.kt

```diff
@@ -315,6 +317,58 @@ class AmplifyAuthCognitoPluginTest {
         verify(mockResult, times(1)).success(ArgumentMatchers.any<LinkedTreeMap<String, Any>>())
     }
 
+    @Test
+    fun fetchSession_returnsSuccess() {
+        val id: AuthSessionResult<String> = AuthSessionResult.success("id")
+        val awsCredentials: AuthSessionResult<AWSCredentials> = AuthSessionResult.success(BasicAWSCredentials("access", "secret"))
+        val userSub: AuthSessionResult<String> = AuthSessionResult.success("sub")
+        val tokens: AuthSessionResult<AWSCognitoUserPoolTokens> = AuthSessionResult.success(AWSCognitoUserPoolTokens("access", "id", "refresh"))
+        val mockSession = AWSCognitoAuthSession(
+        true,
+            id,
+            awsCredentials,
+            userSub,
+            tokens
+        )
+
+        // Arrange
+        doAnswer { invocation: InvocationOnMock ->
+            plugin.prepareCognitoSessionResult(mockResult, mockSession)
+            null as Void?
+        }.`when`(mockAuth).fetchAuthSession(ArgumentMatchers.any<Consumer<AuthSession>>(), ArgumentMatchers.any<Consumer<AuthException>>())
```

Comment:
In test code, `any` is a good candidate for static import, IMO.

Reply:
@jamesonwilliams I was made this change.